### DISCUSSION TITLE
Practice Today

### DISCUSSION THREAD

**Discussion by MikasaSucasa** [frc5567/2023-Robot#165] [Team Posts]
Alright, for practice today, we need to do a lot. Here is just a random working list of things to do
1. Push Comp code to master
2. Review code
3. Talk about what was wrong with it
3a. Height for pickup is slightly low (why)
3b. Figure out climber pwm control (it worked super well so how can we incorporate a stop at a height)
4. Find angle of rocket ship relative to the driver station
5. Re-plug in encoder cables to test bot
6) re-wire test bot

**Comment by MikasaSucasa** [comment]
Hatch pickup device should be made

**Comment by coverbeek** [comment]
- [x] Push Comp code to master
- [x] Review code
    - [ ] Height for pickup is slightly low (why)
    - [ ] Figure out climber pwm control (it worked super well so how can we incorporate a stop at a height)

- [ ] Find angle of rocket ship relative to the driver station
- [ ] Re-plug in encoder cables to test bot
- [ ] re-wire test bot
- [x] Get started on auton
    - [x] Pseudo-code pathing for two-hatch rocket auton on one side
    - [ ] Capture and document on GitHub pseudo-code and next steps (@JooshO  and @FallenFlame5567 -- please make this happen!!!)
    - [ ] Begin implementation with focus on improving hatch-pickup cycle time

**Comment by coverbeek** [comment]
Anyone who missed my urging to watch this video and take a look through the attached slides and looking at the published code on the team 254 GitHub -- you need to watch this video and take a look through the attached slides and look at the published code on the team 254 GitHub....
https://github.com/orgs/frc5567/teams/controls/discussions/20
